sql/src/test/java/io/crate/testing/SqlExpressions.java

```diff
@@ -63,6 +63,10 @@
     private final EvaluatingNormalizer normalizer;
     private final Functions functions;
 
+    public SqlExpressions() {
```

Comment:
Generally I'm not a fan of too many convenience constructors, especially if they are not useful for the common case. Could consider creating `SqlExpressions` just once as private attribute in `ExpressionAnalyzerTest`, that would save the typing overhead as well =)

Reply:
Ay, you're right that too many constructors can be noisy.